libafl/src/executors/command.rs

```diff
@@ -264,6 +277,9 @@ where
         args: IT,
         observers: OT,
         debug_child: bool,
+        has_stdout_observer: Option<String>,
```

Comment:
Why not give it a ref to the observer here?

Reply:
And name it just `stdout_observer`

Reply:
Sorry, you mean a ref to the actual `StdOutObserver` or to the `std::string::String` ?

Reply:
We do that in other places of the code at least, but haven't through it through for this specific instance

Reply:
you cannot have refs to observers around, we use names for that. this code is mine btw iirc, when the CI is green let's merge it and eventually improve it later

Reply:
you can take a ref and use the name from that though

Reply:
Just like feedback does it